Skype/UCWA/PreferredAvailability_ref.md

```diff
@@ -1,4 +1,6 @@
-
+---
+title:  PreferredAvailability for user's availability values 
```

Comment:
Change to "PreferredAvailability reference"

Reply:
done!